src/mumble/AudioConfigDialog.cpp

```diff
@@ -652,6 +623,40 @@ AudioOutputDialog::AudioOutputDialog(Settings &st) : ConfigWidget(st) {
 	qcbLoopback->addItem(tr("Server"), Settings::Server);
 
 	qcbDevice->view()->setTextElideMode(Qt::ElideRight);
+
+	// Distance in cm
+	qsMinDistance->setRange(0, 200);
+	// Distance in m
+	qsbMinimumDistance->setRange(0.0, 50.0);
+	// Distance in cm
+	qsMaxDistance->setRange(10, 2000);
+	// Distance in m
+	qsbMaximumDistance->setRange(1.0, 1000.0);
+	qsMinimumVolume->setRange(0, 100);
+	qsbMinimumVolume->setRange(qsMinimumVolume->minimum(), qsMinimumVolume->maximum());
+	qsBloom->setRange(0, 75);
+	qsbBloom->setRange(qsBloom->minimum(), qsBloom->maximum());
+
+	QString minDistanceTooltip = tr("Distance at which audio volume from another player starts decreasing");
+	QString maxDistanceTooltip = tr("Distance at which a player's audio volume has reached its minimum value");
+	QString minVolumeTooltip =
+		tr("The minimum volume a player's audio will fade out to with increasing distance. Set to 0% for it to fade "
+		   "into complete silence for a realistic maximum hearing distance.");
+	QString bloomTooltip = tr("If an audio source is close enough, blooming will cause the audio to be played on all "
+							  "speakers more or less regardless of their position (albeit with lower volume)");
```

Comment:
@Kissaki I think this description is not too bad. Wdyt?

Reply:
I would specify that audio is played on speakers that are virtually near the source.

For example: if you have a 4.0 surround system (2 front and 2 rear speakers) and the source is at your right in the game, blooming causes audio to be played from the left speaker as well.

Reply:
I first wanted to but all my attempts made tge tooltip become overly large. Do you have something short and concise in mind?

Reply:
"Blooming regulates the audio emitter radius: the higher the value, the more the chance that surrounding speakers are going to reproduce the sound (albeit with lower volume)"

Reply:
I have the feeling that talking about radii here will not be too helpful without an image... :thinking:
Now that I know what this is about, I know and understand what this description is referring to, but when you told that to me for the first time, I did not understand it.

Reply:
Understandable. Maybe we should write a page on our website and leave a link to it in the tooltip.

Reply:
I think links in tooltips don't work well. We can write a doc on the website nonetheless though :)

Reply:
I would indeed prefer tooltips to explain without the need for internet access.